py2many/toposort_modules.py

```diff
@@ -7,7 +7,7 @@
 
 def module_for_path(path: Path) -> str:
     # strip out .py at the end
-    return ".".join(path.parts)[:-3]
```

Comment:
This doesn't do the same thing as before:

```
>>> module_for_path(Path("a/b/c.py"))
'a.b.c'
>>> module_for_path_after(Path("a/b/c.py"))
'a/b/c'
```

Reply:
curious that there were no tests failing.